exasol-s3/src/main/java/com/exasol/spark/s3/S3ClientFactory.java

```diff
@@ -66,7 +66,7 @@ private void setEndpointOverrideIfEnabled(final S3BaseClientBuilder<?, ?> builde
 
     private String getEndpointOverride() {
         final String protocol = getProtocol();
-        return protocol + "://s3." + this.options.get(Option.S3_ENDPOINT_OVERRIDE.key());
+        return protocol + "://" + this.options.get(Option.S3_ENDPOINT_OVERRIDE.key());
```

Comment:
Do we need to document this change in the user guide?

Reply:
Ohh this one is a mistake I think, good catch! 

I made this change since it was not working in vpn environment (domain are resolved on remote resolver). I will revert it.